<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectBrowser</class>
 <widget class="QWidget" name="ProjectBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="filterText">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearFilterBn">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageUnfiltered">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ProjectBrowser::View" name="fileView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFiltered">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ProjectBrowser::View" name="fileViewFiltered"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectBrowser::View</class>
   <extends>QTreeView</extends>
   <header>View.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
